moderators.md

```diff
@@ -0,0 +1,137 @@
+
+As a [moderator](governance.md#moderator), you may optionally restrict your review to packages in your area of expertise.
+Simply search [contributions](https://github.com/r-multiverse/contributions/pulls) for relevant keywords, including [R-multiverse topics](https://r-multiverse.org/topics/).
+
+## Topics
+
+We encourage moderators to contribute [R-multiverse topics](https://r-multiverse.org/topics/) about their fields of expertise.
+To do so, submit a pull request to <https://github.com/r-multiverse/topics> to add a text file describing the subject matter ([example here](https://github.com/r-multiverse/topics/blob/main/bayesian)).^[Through triage access, R-multiverse [moderators](governance.md#moderator) are authorized to merge these pull requests.
+Please only edit the plain files without file extensions.
+All the HTML files are automatically generated.]
+A bot periodically rebuilds the website based on these text files.
+Seasoned [contributors](governance.md#contributor) will see your topic [listed here](https://r-multiverse.org/topics/) and add keywords to their [contributions](https://github.com/r-multiverse/contributions/pulls) accordingly.
+
+## Discussions
+
+[Moderators](governance.md#moderator) help answer questions on the [`r-multiverse/help`](https://github.com/r-multiverse/help) [issue](https://github.com/r-multiverse/help/issues) and [discussion](https://github.com/r-multiverse/help/discussions) forums.
+Questions may ask about specific packages, package reviews, the workings of R-multiverse, etc.
+A [moderator](governance.md#moderator) may close an [issue](https://github.com/r-multiverse/help/issues) or [discussion](https://github.com/r-multiverse/help/discussions) thread if it originally came from a [contributor](governance.md#contributor) or [user](governance.md#user) and the original issue is resolved.
+[Moderators](governance.md#moderator) should not close issues or discussions created by [administrators](team.md#administrators), including [official proposals](governance.md#proposals) to modify [R-multiverse policies](policies.md).
+
+## Enforcement
+
+[Moderators](governance.md#moderator) must understand [all R-multiverse policies](policies.md) and be prepared to enforce them.
+[Moderators](governance.md#moderator) must respond immediately and quickly to the following rare cases.
+Each incident, the actions taken to respond to the incident, and the resolution must be reported as an issue at <https://github.com/r-multiverse/help/issues> with the label "policy violation".
+If you do not feel comfortable taking direct action yourself, contact an [administrator](team.md#administrators) immediately and ask the [administrator](team.md#administrators) handle the situation.
+
+1. In case of an extreme policy violation, or repeated violations of the [code of conduct](conduct.md),
```

Comment:
Is there a code of conduct committee? Is the CoC also reinforced by moderators? There is a process for reinforcing the CoC? How do moderators know when someone violates the CoC more than once?

Reply:
b3973ec8d849d0c4bc38a0ebe379560a4441ef08 references the enforcement policy in the code of conduct. Beyond that, I think CoC enforcement and general policy enforcement both need more planning. I will open an issue at https://github.com/r-multiverse/help/issues.

Reply:
Posted https://github.com/r-multiverse/help/issues/117

Reply:
In a meeting today, @maelle, @shikokuchuo, @jeroen, and I agreed to work on the CoC and committee in a separate issue. We agreed that CoC-related matters are out of the scope of the moderator role (other than reporting them privately to https://github.com/r-multiverse/moderation).